airbyte-cdk/python/unit_tests/sources/file_based/test_scenarios.py

```diff
@@ -227,7 +234,7 @@ def _verify_expected_logs(logs: List[Dict[str, Any]], expected_logs: List[Dict[s
 
 
 @pytest.mark.parametrize("scenario", spec_scenarios, ids=[c.name for c in spec_scenarios])
-def test_spec(capsys, scenario):
+def test_spec(capsys: CaptureFixture[str], scenario: TestScenario) -> None:
```

Comment:
not sure what happened on the prior test run that failed prior to running the cdk tests, but I would expect this test to fail because we're modifying the configs w/ the `filetype` literal and the mapping -> Union. So we'll also have to modify this test scenario with the updated spec.

Reply:
you're right. updated the [expected spec](https://github.com/airbytehq/airbyte/pull/28320/files#diff-261d057402b704966dbd561ac054ecb582646d2e5cdef5ee57ef81d6bfc32e62R43-R254)